lib/SPIRV/libSPIRV/SPIRVEnum.h

```diff
@@ -43,16 +43,23 @@
 #include "LLVMSPIRVOpts.h"
 #include "SPIRVOpCode.h"
 #include "spirv.hpp"
+#include <climits>
```

Comment:
I believe we don't need this header anymore, right?

Reply:
Yes, sure. Thank you.